components/automate-ui/src/app/pages/service-groups/service-groups.component.ts

```diff
@@ -75,7 +75,7 @@ export class ServiceGroupsComponent implements OnInit, OnDestroy {
   filtersVisible = true;
 
   // autocomplete suggestions
-  nodeSuggestions$: Observable<any[]>;
+  serviceGroupsSuggestions$: Observable<any[]>;
```

Comment:
should these all observable string arrays?

Reply:
Yes, I would second the suggestion to get rid of `any` for a specific type. (If you search through your code changes, wherever you reference suggestions it uses `any`, so when changing here, please check for the rest, too.) Thx!